ConventionMobile/ConventionMobile/GlobalVars.cs

```diff
@@ -535,31 +562,40 @@ public static string GenEventAllEventsCustomizableURL
             }
         }
 
+        ///// <summary>
+        ///// Convenience URL to retrieve count of all events
+        ///// </summary>
+        //public static string GenEventAllEventsCountURL
+        //{
+        //    get
+        //    {
+        //        return String.Format(GenEventAllEventsCustomizableURL, yearlyStartingDate.ToString("yyyy-MM-dd't'HH:mm:ss")) + getOption<string>("GenEventAllEventsCountURL", "/numResults");
+        //    }
+        //    set
+        //    {
+        //        setOption("GenEventAllEventsCountURL", value);
+        //    }
+        //}
+
```

Comment:
not a fan of commented code (as you know)  lets kill it unless you feel its needed to keep for some reason.

Reply:
Agreed, I forgot to remove it as part of the update, we can just delete